### DISCUSSION TITLE
m2m: option to  hide relational item in drawer

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#22419] [Feature Requests]
### Summary

The m2m drawer always includes the relational field as well as the additional fields from the junction table. 

In some use-cases we might want to display the junction fields only (e.g when using the m2m for a translation).

### Motivation

Manage junction fields only from the drawer (e.g in translational setups)

### Detailed Design

Therefore an additional prop `displayRelatedItem`
Would need to be added to`drawer-item` :

https://github.com/directus/directus/blob/main/app/src/views/private/components/drawer-item.vue#L406-L432

and than being used in the interface:
https://github.com/directus/directus/blob/1d7e0b71606bb5eb7cce86d810a72bfa9f7b6f6b/app/src/interfaces/list-m2m/list-m2m.vue#L591-L602

### Must Have:
- extend drawerItem with a new prop
- update usage

### Drawbacks

The related item might be needed in the editing context, but don’t think so. If so, it could simply be hidden based on the prop.

### Alternatives

N/a

### Adoption Strategy

N/a

**Comment by kegesch** [comment]
Seems to relate to: https://github.com/directus/directus/discussions/19040
